jpos/src/main/java/org/jpos/space/JDBMSpace.java

```diff
@@ -237,7 +238,7 @@ public void push (Object key, Object value, long timeout) {
             synchronized (this) {
                 long recid = recman.insert (value);
                 long expiration = timeout == -1 ? Long.MAX_VALUE :
-                        Instant.now().toEpochMilli() + timeout;
+                        Instant.now(NanoClock.systemUTC()).toEpochMilli() + timeout;
```

Comment:
I believe here the use of `NanoClock`can be justified

Reply:
For persisted objects, the timeouts are usually measured in minutes, hours, or no timeout. Not sure we need nano precision here.

Reply:
Well I'm trying to ensure we have monotonic behavior here as much as possible, hence the use of `NanoClock`.